src/actions/expressions.js

```diff
@@ -37,26 +38,30 @@ export function addExpression(input: string) {
       return dispatch(evaluateExpression(expression));
     }
 
-    dispatch({
-      type: "ADD_EXPRESSION",
-      input
-    });
+    dispatch({ type: "ADD_EXPRESSION", input, expressionError });
```

Comment:
I think this makes sense, but just confirming that we can add an expression w/ an error